src/irmin-unix/info.mli

```diff
@@ -14,8 +14,11 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
-module Make (I : Irmin.Info.S) : sig
-  include Irmin.Info.S with type t = I.t
+module type S = sig
+  include Irmin.Info.S
 
-  val v : ?author:string -> ('b, Format.formatter, unit, f) format4 -> 'b
+  val vf : ?author:string -> ('b, Format.formatter, unit, f) format4 -> 'b
```

Comment:
I wasn't sure about naming here, but I didn't want to override the `v` in `Irmin.Info` (and the compiler didn't like it since types didn't match for this to stand in as `Irmin.Info.S`). I appended `f` for `formatter`.